app/dependencies.php

```diff
@@ -502,6 +502,7 @@ static function (ContainerInterface $c): Matching\Adapter {
         },
 
         ClockInterface::class => fn() => new NativeClock(),
+        Psr\Clock\ClockInterface::class  => fn() => new NativeClock(),
```

Comment:
took me a while to work out why I was getting an error about ClockInterface not being instantiate despite line 504 already being there before I worked out it was because we only put the Symfony interface in here not the PSR interface. Arguably we should stick to one for consistency but it doesn't do any harm using the PSR one when we don't need the features from Symfony, and especially since they have the same name its a somewhat confusing error if we don't have the PSR one defined.

Also there's some inconsistency because in some services we inject a clock and in others we inject just a `$now` \DateTime.